pkg/kopia/utils.go

```diff
@@ -184,3 +198,35 @@ func GetDataStoreGeneralMetadataCacheSize(opt map[string]int) int {
 	}
 	return defaultDataStoreGeneralMetadataCacheSizeMB
 }
+
+// GetCustomConfigFileAndLogDirectory returns a config file path and log directory based on the hostname
+func GetCustomConfigFileAndLogDirectory(hostname string) (string, string) {
+	hostname = strings.ReplaceAll(hostname, ".", "-")
+	configFile := filepath.Join(kopiacmd.DefaultConfigDirectory, hostname+".config")
+	logDir := filepath.Join(kopiacmd.DefaultLogDirectory, hostname)
+	return configFile, logDir
+}
+
+// WaitTillCommandSucceed returns error if the Command fails to pass without error before default timeout
+func WaitTillCommandSucceed(ctx context.Context, cli kubernetes.Interface, cmd []string, namespace, podName, container string) error {
+	err := poll.WaitWithBackoff(ctx, backoff.Backoff{
```

Comment:
woudl `Wait` from `poll.go` not work here?

Reply:
I think this function is unused, will remove it !